thunderbird/processes/wps_split_merged_climos.py

```diff
@@ -0,0 +1,118 @@
+# Processor imports
+from pywps import (
+    Process,
+    LiteralInput,
+    ComplexInput,
+    ComplexOutput,
+    FORMATS,
+)
+from pywps.app.Common import Metadata
+
+# Tool imports
+from nchelpers import CFDataset
+from dp.split_merged_climos import split_merged_climos
+from thunderbird.utils import (
+    MAX_OCCURS,
+    is_opendap_url,
+    collect_output_files,
+    build_meta_link,
+)
+
+# Library imports
+import logging
+import os
+import math
+
+logger = logging.getLogger("PYWPS")
+
+
+class SplitMergedClimos(Process):
+    def __init__(self):
+        self.log_level_choices = [
+            "NOTSET",
+            "DEBUG",
+            "INFO",
+            "WARNING",
+            "ERROR",
+            "CRITICAL",
+        ]
```

Comment:
Probably overkill since this is clear enough. But you could use `logging._levelToName` to get all the level names you need.